cms/views_test.py

```diff
@@ -51,11 +51,6 @@ def test_home_page_view(client):
     resp = client.get(page.get_url())
     content = resp.content.decode("utf-8")
 
-    # with watch now button
-    assert (
-        f'<a id="actionButton" class="btn btn-primary text-uppercase px-5 py-2 action-button" href="#">Watch Now</a>'
-        in content
-    )
```

Comment:
why needs to remove this? this is already decreasing coverage.

Reply:
We removed the Enroll Now button from Hero section.

Reply:
Revert this when you restore the enroll now button in the hero section.

Reply:
even if we are deleting it, you can update this test to validate that a button is appearing in navbar.